.github/workflows/integration-tests.yml

```diff
@@ -622,11 +938,17 @@ jobs:
       - name: Build Go Test Command
         id: build-go-test-command
         run: |
+          # if dir is provided use it, otherwise use the smoke dir
+          if [ "${{ matrix.product.dir }}" != "" ]; then
+            dir=${{ matrix.product.dir }}
+          else
+            dir=smoke
+          fi
           # if the matrix.product.run is set, use it for a different command
           if [ "${{ matrix.product.run }}" != "" ]; then
-            echo "run_command=${{ matrix.product.run }} ./smoke/${{ matrix.product.file }}_test.go" >> "$GITHUB_OUTPUT"
+            echo "run_command=${{ matrix.product.run }} ./${dir}/${{ matrix.product.file }}_test.go" >> "$GITHUB_OUTPUT"
           else
-            echo "run_command=./smoke/${{ matrix.product.name }}_test.go" >> "$GITHUB_OUTPUT"
+            echo "run_command=./${dir}/${{ matrix.product.name }}_test.go" >> "$GITHUB_OUTPUT"
```

Comment:
in the future will we want to move ccip tests to the same folder as other integration tests?

Reply:
Yes, efforts there are tracked in TT-1441